server/game/cards/06-WoE/Bryozoarch.js

```diff
@@ -0,0 +1,25 @@
+const Card = require('../../Card.js');
+
+class Bryozoarch extends Card {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            targetController: 'opponent',
+            match: (card) => card.type === 'action',
+            effect: ability.effects.blank()
```

Comment:
I am not sure I like to use blank. I think we should have an effect that says `cannot resolve`

Reply:
For example, how it will interact with "High Priest Torvus" (Thank Lord this card is not in WoE) :)

Reply:
Does the new version look better?  I'm not sure I fully understand how all this works.

Reply:
I guess not, because I believe the interrupt you added you fizzle also the enhancements on the card played

Reply:
The test makes sure that the opponent gets the amber bonus.  Is the test wrong?

Reply:
ok, so probably the amber is somewhere else. It would be nice to add a test with High Priest Torvus on player1 field. It should not trigger its effect, but here I think the problem is in High Priest Torvus implementation. If it triggers, just `xit` the test case so we know about it in the future.

![](https://archonarcana.com/File:496-142.png)

Reply:
Ok added the test case, and you're right, it fails.  `xit`'d it out.  Let me know if you want me to make an issue for it.

Reply:
no need, the xit is enough